apps/foundation-app/src/app/cart/components/cart-summary/cart-summary.component.html

```diff
@@ -0,0 +1,9 @@
+<div class="cart-summary">
+  <div class="cart-summary__title">cart summary</div>
+  <promotion></promotion>
+  <cart-subtotal [cart]="cart"></cart-subtotal>
```

Comment:
I'd say each of these should be a `cart-total`, I don't think we need a unique component for each of these. I think the API for a `cart-total` is `title` and `value`.

Reply:
Ok, so I made this into two components, `cart-subtotal` and `cart-grand-total`, which both implement a common interface, `cart-total`. The only difference is that `cart-grand-total` has some blue font.